debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/history/JdbcSchemaHistory.java

```diff
@@ -116,7 +116,7 @@ protected void storeRecord(HistoryRecord record) throws SchemaHistoryException {
                     sql.setInt(3, partSeq);
                     sql.setTimestamp(4, currentTs);
                     sql.setInt(5, recordInsertSeq.incrementAndGet());
-                    sql.executeUpdate();
+                    sql.execute();
```

Comment:
Why is this change needed?

Reply:
reverted change.

Reply:
will cover this in another PR for mysql.